apps/dialog/src/db/seed/federal-state.ts

```diff
@@ -28,6 +28,8 @@ export async function insertFederalStates({ skip = true }: { skip: boolean }) {
       federalStateId: federalStateAndApiKey.id,
     });
   }
+
+  console.info('federalState seed successful');
```

Comment:
minor, other places use `log` instead of `info`
```suggestion
  console.log('federalState seed successful');
```